.gitignore

```diff
@@ -127,4 +127,8 @@ runConfigurations.xml
 # Mac OS X
 .DS_Store
 
-reports/
\ No newline at end of file
+reports/
+
+
+#VSCode
+.vscode/launch.json
```

Comment:
Did you add custom run configs or why did you have changed `launch.json` files? If those launch configs make sense I think we could just include them in the repo/commit those changes.